pkg/synopsysctl/soperator_resources.go

```diff
@@ -274,3 +277,86 @@ func (specConfig *SOperatorSpecConfig) GetOperatorClusterRoleBinding() *horizonc
+		APIVersion: "rbac.authorization.k8s.io/v1beta1",
+		//ClusterName : "string,"
+		Name:      "synopsys-operator-admin",
+		Namespace: specConfig.Namespace,
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+		APIGroups:       []string{"apiextensions.k8s.io"},
+		Resources:       []string{"customresourcedefinitions"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+		APIGroups:       []string{"rbac.authorization.k8s.io"},
+		Resources:       []string{"clusterrolebindings", "clusterroles"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+		APIGroups:       []string{""},
+		Resources:       []string{"namespaces", "configmaps", "persistentvolumeclaims", "services", "secrets", "replicationcontrollers", "deployments", "statefulsets", "serviceaccounts"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch"},
+		APIGroups:       []string{""},
+		Resources:       []string{"pods", "pods/log", "endpoints"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"create"},
+		APIGroups:       []string{""},
+		Resources:       []string{"pods/exec"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+		APIGroups:       []string{"synopsys.com"},
+		Resources:       []string{"*"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	// Add Openshift rules
+	restConfig := util.GetKubeRestConfig()
+	routeClient, err := routeclient.NewForConfig(restConfig) // kube doesn't have a routeclient
+	if routeClient != nil && err == nil {                    // openshift: have a routeClient and no error
+		synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+			Verbs:           []string{"get", "update", "patch"},
+			APIGroups:       []string{"security.openshift.io"},
+			Resources:       []string{"securitycontextconstraints"},
+			ResourceNames:   []string{},
+			NonResourceURLs: []string{},
+		})
+
+		synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+			Verbs:           []string{"get", "create"},
```

Comment:
Probably want a full set of verbs here so we can delete/update/modify any routes we create.

Reply:
Updated to a full set of verbs.